docs/http/http-spans.md

```diff
@@ -61,13 +61,20 @@ and various HTTP versions like 1.1, 2 and SPDY.
 ## Name
 
 HTTP spans MUST follow the overall [guidelines for span names](https://github.com/open-telemetry/opentelemetry-specification/tree/v1.22.0/specification/trace/api.md#span).
-HTTP server span names SHOULD be `{http.request.method} {http.route}` if there is a
-(low-cardinality) `http.route` available.
-HTTP server span names SHOULD be `{http.request.method}` if there is no (low-cardinality)
-`http.route` available.
+
+HTTP server span names SHOULD be `{method} {http.route}` if there is a
+(low-cardinality) `http.route` available,
+
+If there is no (low-cardinality) `http.route` available, HTTP server span names
+SHOULD match `{method}`
```

Comment:
```suggestion
HTTP server span names SHOULD be `HTTP {method} {http.route}` if there is a
(low-cardinality) `http.route` available,

If there is no (low-cardinality) `http.route` available, HTTP server span names
SHOULD match `HTTP {method}`
```

Reply:
Just thinking out loud: Would such change now be considered a breaking change for users? Do people usually rely on the names to build custom alerts or such things? 🤔

Reply:
Doesn't that reverse https://github.com/open-telemetry/opentelemetry-specification/pull/3165 though?

Reply:
@mateuszrzeszutek  Yes it does revert that, but not only.  The reason for the HTTP removal IIUC was to keep route-based and method-based span names consistent, which is achieved here by adding the HTTP prefix also for route-based span names.